.github/workflows/gcovr-ci-job.yml

```diff
@@ -119,10 +124,9 @@ jobs:
         run: |
           nox --non-interactive $NOX_CONTAINER_ARGUMENTS --session tests --  --archive_differences
           # There must not be any modifications to get the correct versions later
-          Status=$(git status --porcelain)
-          if [ -n "$Status" ] ; then
+          if [ -n "$(git status --porcelain)" ] ; then
             echo "Following files are modified:"
-            echo "$Status"
+            git diff
```

Comment:
`git diff --name-status` 

if you only want to see which files were changed without the full diff in each one.

Reply:
I want to see the differences here because on local system there was no change. The diff pointed out that the version 0.0+main was used in CI because the tags are not fetched anymore (see https://github.com/actions/checkout/issues/2041).